index.js

```diff
@@ -0,0 +1,41 @@
+const fs = require('fs')
+// console.log(__dirname)
+const path = require("path")
+
+const languages = ["r", "python", "node.js"]
+
```

Comment:
We'll probably want to run `prettier` over this. I'll go ahead and submit another PR. 😄